haproxy.cfg

```diff
@@ -32,29 +32,22 @@ frontend http_front
     acl host_equitytool req.hdr(host) -i -m beg equitytool.austinmobility.io
     redirect scheme https code 301 if host_equitytool
 
-    # PostgREST Path ACLs
-    acl path_knack_services path_beg /knack-services
-    acl path_legacy_scripts path_beg /legacy-scripts
-    acl path_ctr_data_lake path_beg /ctr-data-lake
-    acl path_parking path_beg /parking
-    acl path_road_conditions path_beg /road-conditions
-    acl path_bond_reporting path_beg /bond-reporting
-
-    # Host and source ACLs
+    # Only go to postgrest if the path is valid AND:
+    #   - either source is 10.0.0.0/8
+    #   - or source is 162.89.100.0/24 and host header is present
+
+    acl path_postgrest path_beg /knack-services /legacy-scripts /ctr-data-lake /parking /road-conditions /bond-reporting
```

Comment:
This new ACL line matches any of these path beginnings, see https://docs.haproxy.org/3.0/configuration.html#7.2.